src/modules/dataTypes.js

```diff
@@ -18,6 +18,8 @@ export const DIMENSION_TYPE_PERIOD = 'PERIOD'
 export const DIMENSION_TYPE_ORGANISATION_UNIT = 'ORGANISATION_UNIT'
 export const DIMENSION_TYPE_ORGANISATION_UNIT_GROUP_SET =
     'ORGANISATION_UNIT_GROUP_SET'
+export const DIMENSION_TYPE_EXPRESSION_DIMENSION_ITEM =
```

Comment:
Should this be exported from `src/index/js` like all the others?

Reply:
Yes it can be exported. I guess it's not already since it's not used by any of the apps. But I've added the export now 👍